packages/dds/tree/src/simple-tree/toStoredSchema.ts

```diff
@@ -90,7 +90,7 @@ export function convertField(schema: SimpleFieldSchema): TreeFieldStoredSchema {
 	const kind: FieldKindIdentifier =
 		convertFieldKind.get(schema.kind)?.identifier ?? fail(0xae3 /* Invalid field kind */);
 	const types: TreeTypeSet = schema.allowedTypesIdentifiers as TreeTypeSet;
-	return { kind, types };
+	return { kind, types, metadata: schema.persistedMetadata };
```

Comment:
So if I'm getting this right, in the StoredSchema layer we're calling it plain `metadata` because it's the only one that we persist so `persistedMetadata` would be redundant (and as Craig pointed out at some point, wasteful because the property name consumes actual bytes on the wire and when stored at rest), and when moving to the SimpleSchema layer is when we map it into `persistedMetadata` to differentiate it from the transient one that already exists in the SimpleSchema domain, correct?

Reply:
If so, maybe we could benefit from `@privateremarks` on the relevant properties on both sides to clarify it for people who start working in this part of the codebase.